caim_base/templates/user_profile/view.html

```diff
@@ -1,37 +1,60 @@
-{% extends 'base/wrapper.html' %}
+{% extends "base/wrapper.html" %}
 {% load avatar_tags %}
-
 {% block content %}
-<div class="bg-dark context-bar">
+  <div class="bg-dark context-bar">
     <div class="container">
-      <div class="row">
-        <div class="col-md-6 py-3 d-none d-md-block">
-            <p class="text-light my-0">
-              &nbsp;
-            </p>
-          </div>
-        <div class="col-md-6 py-2 text-center text-md-end d-bottom-mobile">
-            {% if user == request.user %}
-                <a href="/user/{{ user.username }}/edit" class="btn btn-secondary">Edit profile</a>
-                <a href="{% url 'avatar_change' %}" class="btn btn-secondary">Change profile image</a>
-            {% endif %}
+      <div class="d-flex justify-content-center justify-content-sm-end">
+        <div class="text-center text-md-end d-flex flex-column flex-sm-row gap-2 py-2">
```

Comment:
Ok so something that's been confusing me a little bit. This looks an awful lot like Tailwind CSS, but I don't see django-tailwind installed anywhere, and we're not running the tailwind "compiler" runserver command as part of the development processes. Could someone who knows a little more about CSS than I do explain what's going on here?

Reply:
its boostrap5, which is heavily inspired by tailwind. https://getbootstrap.com/docs/5.0/getting-started/introduction/

No need for CSS compiling, as they are just CSS rules.